cli/cmd/plugin/ui/web/tanzu-ui/src/shared/services/awsDeployment.tsx

```diff
@@ -40,7 +41,7 @@ const useAwsDeployment = () => {
             createCloudFormationStack: false,
             clusterName: awsData[AWS_FIELDS.CLUSTER_NAME],
             controlPlaneFlavor: awsData[AWS_FIELDS.CLUSTER_PLAN],
-            controlPlaneNodeType: retrieveAwsInstanceType(awsData[AWS_FIELDS.NODE_PROFILE]),
+            controlPlaneNodeType: nodeType[awsData[AWS_FIELDS.NODE_PROFILE]],
```

Comment:
Just to confirm, the controlPlaneNodeType is getting set to an AWS instance type here right? Examples would be 't3a.large' or 'm6a.xlarge'

Reply:
Yes sir. The value of `controlPlaneNodeType` is the AWS instance type.